### ISSUE TITLE
tf _approxmatch_so cannot be used in higher version tensorflow

### ISSUE THREAD

**Issue by GuanlinLee** [ajhamdi/AdvPC#5]
I am using Tensorflow-gpu=1.14.0, CUDA=10.2.
I try to compile the tf_approxmatch_so by myself. However, there is an error when I try to import it.
`
tensorflow.python.framework.errors_impl.NotFoundError: /root/AdvPC/latent_3d_points/external/structural_losses/tf_approxmatch_so.so: undefined symbol: _ZN10tensorflow12OpDefBuilder5InputESs
`
I find others have met this error before, when they compile StyleGAN. But their solutions do not help me solve it. Hope you can do some tests on Tensorflow-gpu=1.14.0, CUDA=10.2, and help me solve it.

BTW, the tensorflow is installed from conda instead of pip. I know they are compiled by different version gcc compiler.

**Comment by ajhamdi** [comment]
@GuanlinLee  Yes this error can happen because the newer TF version does not use this symbol needed for compiling the C++ code. I do not have a solution yet, but will keep looking for one and will update you once I have it. Please try to install TF 1.10 and run the code to make sure it is running and that the TF versions is the issue
